packages/vue-app/template/store.js

```diff
@@ -14,7 +14,7 @@ void (function updateModules() {
   <% return true }}) %>
 
   // If store is an exported method = classic mode (deprecated)
-  if (typeof store === 'function') {
+  if (typeof store === 'function' && process.env.NODE_ENV !== 'production') {
```

Comment:
Better use `<% if (isDev) { %>` so it's not included in the bundle itself ☺️

Reply:
Thank you for your feedback! I fixed it!